pytorch/training/docker/1.8.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,127 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Add arguments to achieve the version, python and url
+ARG PYTHON_VERSION=3.6.10
```

Comment:
Python version should be upgraded to 3.6.13.

Additionally, similar changes to the one made in https://github.com/aws/deep-learning-containers/pull/860 probably apply.

This can wait until you have a successful build on the PR, though.